src/clj/game/core/engine.clj

```diff
@@ -618,15 +618,16 @@
   This is essentially Phase 9.3 and 9.6.7a of CR 1.1:
   http://nisei.net/files/Comprehensive_Rules_1.1.pdf"
   ([state side event targets] (gather-events state side event targets nil))
-  ([state side event targets card-abilities]
+  ([state side event targets card-abilities] (gather-events state side event targets nil (make-eid state)))
+  ([state side event targets card-abilities eid]
```

Comment:
Even if they're not being completed, `eid`s should be in the third position for consistency sake.